### PR TITLE
Add domain field to Group Schema

### PR DESCRIPTION
**Pull request by janniten**
Added group.domain field in the group schema.
I've been working with Window's Security Groups Events and the group domain needs to be mapped into ECS in order to be able to correlate with others events
(Example events 4731,4732,4733,... where the SubjectDomainName is the domain of the group created/modified/deteled, etc)
Another use case I found to justified the existence of the group,domain was several Fortigate Logs where group="DOMAIN\GROUP" appears as field (for example log_id=0315093008)

### CODE REVIEW

**Comment by webmat** [comment]
Thanks for submitting this!

I think this makes sense, yes. @dainperkins WDYT?

File: schemas/group.yml [hunk 1]
```diff
@@ -26,3 +26,11 @@
       type: keyword
       description: >
         Name of the group.
+
+    - name: domain
+      level: extended
+      type: keyword
+      description: >
```
Comment (webmat) [inline schemas/group.yml]: Please add a `short` description as well, with just the first sentence.

`short: Name of the directory the group is a member of.`

Don't forget to run make prior to committing ;-)

Comment (janniten) [reply schemas/group.yml]: Done! Pls let me know if everything is OK

**Comment by webmat** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by janniten** [comment]
> Alright, this is almost good to go. Two minor changes and I'll merge:
> 
> * Please add a changelog entry in CHANGELOG.next.md
> * Please add an empty line before "For example...", this will ensure they appear as two paragraphs in the docs.
> 
> Thanks again :-)

Done! 
Regards

**Pull request merged**